templates/company/registers/list.html.tx

```diff
@@ -1,4 +1,4 @@
-% cascade base { title => $company.company_name ~" - Registers (free information from Companies House)", require_js => "toggle_links"}
+% cascade base { title => $company.company_name ~" registers - Find and update company information - GOV.UK", require_js => "toggle_links"}
```

Comment:
If this has a different use case to the `templates/company/pscs/list.html.tx`, shouldn't it have a title that differentiates it? - having the same title on multiple pages isn't going to help someone who relies on a screen reader?

Reply:
`templates/company/pscs/list.html.tx` has `{ title=> $company.company_name ~" persons with significant control - Find and update company information - GOV.UK" }`